src/core/Utils.test.js

```diff
@@ -5,16 +5,18 @@ const sampleImageDataURI =
 
 describe('core/utils', () => {
   describe('generateFileID', () => {
-    it('should take the filename object and produce a lowercase file id made up of the name and lastModified date', () => {
+    it('should take the filename object and produce a lowercase file id made up of uppy- prefix, file name (cleaned up to be lowercase, letters and numbers only), type, size and lastModified date', () => {
       const fileObj = {
         name: 'fOo0Fi@£$.jpg',
+        type: 'image/jpeg',
         data: {
-          lastModified: '2017-08-31T00:00:00.000Z'
+          lastModified: '2017-08-31T00:00:00.000Z',
```

Comment:
Tangential nit, `lastModified` is a number in browsers.

Reply:
I don’t know where the date object came from, yeah, it’s a number. Thanks!